HashRegistrarSimplified.sol

```diff
@@ -231,6 +295,11 @@ contract Registrar {
         bid.setOwner(_owner);
         entry h = entries[_hash];
 
+        /* 
+        A penalty is applied for submitting pointless bids. This prevents bids which
+        are never intended to be revealed during the reveal period, which could otherwise
+        appear falsely as an expression of interest.
```

Comment:
The main threat is that a participant could use the threat of revealing a bid that's higher than the second-highest bid to extort the winner into paying them.